CMSgov/design-system
# [WNMGDS-596] Fix for checked checkbox background variable

Checked checkboxes and radios were not using the proper variable. 
This Pr updates to the correct variable.

### How to test
- run the doc site locally and visit the choice page and ensure the proper color is being used `#0071bc`
